test

```diff
@@ -59,3 +59,4 @@ test coroutines-co.js
 test coroutines-bluebird.js
 test await.compiled.js
 test tasks.js
+test kgo.js
```

Comment:
using kgo and async are not very similar approaches, foreign.parallel and async.map are similar.